netbox_cmdb/netbox_cmdb/models/vlan.py

```diff
@@ -34,10 +27,9 @@ class VLAN(ChangeLoggedModel):
     )
 
     def __str__(self):
-        return f"{self.device.name}--{self.name}"
+        return f"{self.name}"
 
     class Meta:
         ordering = ["vid"]
-        unique_together = ("device", "name")
```

Comment:
shouldn't  we keep a unique together between vid and name ?

we could have multiple times the same vid in database, but not with the same name otherwise it will be complex to differentiate them

Reply:
@fdomain why we should not keep device?
I mean, having the list of Vlan per devices is useful. Otherwise, it will be complex to get the list to set it on the device.